### DISCUSSION TITLE
Change the sort order

### DISCUSSION THREAD

**Discussion by bhepple** [minad/vertico#521] [Q&A] [answered]
I see that when vertico is used with consult-find, the choices are presented sorted by the string length of the candidates.

I would prefer them to be sorted alphabetically.

How can that be done?

I can't see anything obvious in the wikis, the source code or the help doco. No doubt my reading has been too shallow.

**Comment by minad** [comment, marked as answer]
You can try `(consult-customize consult-find :sort t)` and configure `vertico-sort-function` as you like, also per-command only for `consult-find` via `vertico-multiform-mode`.
